src/main/kotlin/io/github/flank/gradle/simple-flank.gradle.kts

```diff
@@ -126,7 +128,7 @@ fun registerFlankRun(
     tasks.register<FlankRunTask>("flankRun${variant.name.capitalize()}") {
       flankJarClasspath.from(flankExecutable)
 
-      serviceAccountCredentials.convention(simpleFlankExtension.credentialsFile)
+      serviceAccountCredentials.value(simpleFlankExtension.credentialsFile)
```

Comment:
Why?

Reply:
Because the file might not exist. Indeed, if you look at the `simpleFlankExtension.credentialsFile` it's a little bit complex.
If you set a file that might not exist as convention of a Property, gradle fails.
Today I discovered that in these cases we should use `@InputFiles` instead of `@InputFile` in these cases ([docs](https://docs.gradle.org/current/userguide/validation_problems.html#input_file_does_not_exist)).
I will update the PR with this change.